src/services/Http.hh

```diff
@@ -10,10 +10,16 @@ namespace services {
 class Http
 {
 public:
+    enum class StatusCode
+    {
+        OK = 200,
+        NotFound = 404,
+    };
+
     class Response
     {
     public:
-        explicit Response(int nStatusCode, const std::string& sResponse) noexcept
+        explicit Response(Http::StatusCode nStatusCode, const std::string& sResponse) noexcept
```

Comment:
This might not be able to be `noexcept` because `std::string`'s copy constructor is not `noexcept`.